### ISSUE TITLE
1.12.2 Server error

### ISSUE THREAD

**Issue by Four-Velocity** [Zuxelus/Combo-Armors#3]
Hi! I tried to run Forge server and get this error on connect
```
[22:57:47] [Server thread/WARN] [minecraft/NetworkSystem]: Failed to handle packet for /192.168.0.188:44114
net.minecraft.util.ReportedException: Ticking player
	at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:459) ~[oq.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_73660_a(NetHandlerPlayServer.java:173) ~[pa.class:?]
	at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.func_73660_a(NetworkDispatcher.java:209) ~[NetworkDispatcher$1.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:285) ~[gw.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:180) [oz.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:790) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_275]
Caused by: java.lang.NoSuchMethodError: net.minecraft.entity.player.InventoryPlayer.func_70440_f(I)Lnet/minecraft/item/ItemStack;
	at com.zuxelus.comboarmors.ServerTickHandler.onPlayerTick(ServerTickHandler.java:35) ~[ServerTickHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_540_ServerTickHandler_onPlayerTick_PlayerTickEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) ~[EventBus.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onPlayerPreTick(FMLCommonHandler.java:365) ~[FMLCommonHandler.class:?]
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:190) ~[aed.class:?]
	at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:382) ~[oq.class:?]
```
As i see it can be fixed this way https://forums.minecraftforge.net/topic/65662-ifplayerinventoryarmoriteminslot-crash-on-server/ and i'll try to do it today, but i'm not familiar with Java, so maybe you will do it faster :)

**Comment by Four-Velocity** [comment]
Just to clarify. It happens only with standalone server. When I use "Share to LAN" everything working fine

**Comment by Zuxelus** [comment]
Should be fixed in 1.12.2-0.0.4

**Comment by Four-Velocity** [comment]
Just checked and looks like everything works fine. Thank you, you are the best!
